docs/troubleshooting-guide/permissions.md

```diff
@@ -0,0 +1,50 @@
+# Fixing permissions issues
+
+Metabase operates with two types of [permissions][admin-permissions]: **data permissions** and **collection permissions**. Permissions in Metabase don't operate on an all-or-nothing basis — as an administrator you'll weave together these data and collection permission settings based on the needs of different user groups to form your organization's overall permissions structure. If you're not sure where to start in setting up and managing permissions, check out our [Learn track][learn-permissions] on the subject.
+
+Certain permissions features, like [sandboxing][sandboxing] and blocking database access to groups, are exclusive to [paid plans](https://www.metabase.com/pricing). 
+
+This troubleshooting guide has you covered if you've connected your database to Metabase, set up groups for new users, and granted data and collection permissions to those groups, but:
+
+- users in multiple groups can access data they shouldn't, or
+- users can view collections that contain data you want to restrict, or
+- users who need to can't access the SQL Editor.
+
+## A user in multiple groups can access a table that you want to restrict
+
+**Root cause:** This person is a member of multiple groups, in which case Metabase grants the *most permissive* level of access across all the groups in which they're a member.
+
+If they're a member of two groups — one which grants **Unrestricted** access to a database or table and another that grants **No self-service** access — that user will have full unrestricted access.
+
+**Steps to take:**
+
+1. In the **People tab** of the **Admin Panel**, take a look at which groups the user is in.
+2. Either remove the user from the group with wider permissions, or adjust that group's data permissions to grant them no-self service access to the table you want restricted.
+
+Remember that everyone is a member of the **All Users** group; this is why we recommend you revoke permissions from the **All users** group, and create new groups to selectively apply permissions to your data sources.
+
+## A user can view collections that reference data you want to restrict
```

Comment:
Since the solution here requires money, we should include the plans blockquote to make that clear. We may also want to put this at the bottom of the page, so that as we add more scenarios we can keep the free/paid items grouped.